canonical-web-and-design/microstack.run
# Move to 11ty

## Done

- Replace Jekyll with 11ty
- Fixed Jekyll issues (dotrun not working)
- Now watching JS folder too
- Update dependencies (without Jekyll)
- Updated `dockerFile` and `package.json`

# QA

- Checkout this branch
- `dotrun` or `./run`
- Browse website at http://localhost:8039
- Check all links are working
- Check libraries are working (copy to clipboard, tabs, install version...)
- Make sure checks pass (docker image has been updated too, may need to run it)

Fixes #44
